src/main/kotlin/no/nav/familie/ef/sak/task/BehandlingsstatistikkTask.kt

```diff
@@ -61,28 +61,40 @@ class BehandlingsstatistikkTask(private val iverksettClient: IverksettClient,
 
         val resultatBegrunnelse = finnResultatBegrunnelse(hendelse, vedtak)
         val søker = grunnlagsdataService.hentGrunnlagsdata(behandlingId).grunnlagsdata.søker
-        val søknadstidspunkt = finnSøknadstidspunkt(behandling)
+        val henvendelseTidspunkt = finnHenvendelsestidspunkt(behandling)
 
         val behandlingsstatistikkDto = BehandlingsstatistikkDto(
                 behandlingId = behandlingId,
                 personIdent = personIdent,
                 gjeldendeSaksbehandlerId = finnSaksbehandler(hendelse, vedtak, gjeldendeSaksbehandler),
                 eksternFagsakId = fagsak.eksternId.id.toString(),
                 hendelseTidspunkt = hendelseTidspunkt.atZone(zoneIdOslo),
-                søknadstidspunkt = søknadstidspunkt.atZone(zoneIdOslo),
+                søknadstidspunkt = henvendelseTidspunkt.atZone(zoneIdOslo),
                 hendelse = hendelse,
                 behandlingResultat = behandling.resultat.name,
                 resultatBegrunnelse = resultatBegrunnelse,
                 opprettetEnhet = sisteOppgaveForBehandling.opprettetAvEnhetsnr ?: "9999",
                 ansvarligEnhet = sisteOppgaveForBehandling.tildeltEnhetsnr ?: "9999",
                 strengtFortroligAdresse = søker.adressebeskyttelse?.erStrengtFortrolig() ?: false,
                 stønadstype = StønadType.valueOf(fagsak.stønadstype.name),
-                behandlingstype = BehandlingType.valueOf(behandling.type.name)
+                behandlingstype = BehandlingType.valueOf(behandling.type.name),
+                henvendelseTidspunkt = henvendelseTidspunkt.atZone(zoneIdOslo),
+                relatertBehandlingId = finnRelatertBehandlingId(behandling, hendelse)
         )
 
         iverksettClient.sendBehandlingsstatistikk(behandlingsstatistikkDto)
     }
 
+    private fun finnRelatertBehandlingId(behandling: Behandling, hendelse: Hendelse): UUID? {
+        if (hendelse == Hendelse.FERDIG) {
```

Comment:
Hvorfor trengs denne?

Reply:
Vi leter etter relatert behandling (altså den forrige behandlingen)

finnSisteIverksatteBehandling vil vel finne denne behandlingen hvis den er ferdig? Litt rart å ha A.relatertbehandling = A.behandling?

Reply:
Ah ja 👍 

En sak som jeg lurer på, som er litt Off topic fra denne PR - trenger vi å hente alle disse ulike verdiene for alle typer statistikk? Sånn som statistikk er i iverksett så kopieres mye av dataen mellom hver hendelsetype, https://github.com/navikt/familie-ef-iverksett/blob/main/src/main/kotlin/no/nav/familie/ef/iverksett/behandlingsstatistikk/BehandlingsstatistikkService.kt#L62-L100

Sånn att vi sender unødvendig mye data/henter veldig mye data som ikke brukes?

Reply:
Eller ja, det er ju litt relatert til denne endring, fordi hvis man hadde laget en egen DTO for de andre typene, eks FERDIG, så hadde man ikke trengt å sende med forrigeBehandlingId, ettersom den kun trengs i første meldingen?

Reply:
True.
> trenger vi å hente alle disse ulike verdiene for alle typer statistikk?
Det er sant. Her kan vi kanskje ta en ny runde.